scipy/stats/tests/test_distributions.py

```diff
@@ -2092,6 +2092,62 @@ def test_logcdf_logsf(self):
         logsf = stats.invgauss.logsf(110, 1.05)
         assert_allclose(logsf, -56.1467092416426)
 
+    def test_pdf(self):
+        # regression tests for improvements introduces in gh-13665
+        # these tests are bases on Gyner & Smyth (2016)
+
+        # infinite mean corresponds to an inverse-chisquare pdf.
+        assert_allclose(
+            stats.invgauss.pdf([-1, 0, 1, 2, np.inf, np.nan], mu=np.inf),
+            [0., 0., 0.24197072, 0.10984782, 0., np.nan],
```

Comment:
I can't find the non-trivial values are not in the paper(although I confirmed them with https://rdrr.io/cran/statmod/). What is the source?

Reply:
This is a regression test related to #13665 to test the return values for non-trivial input when the mean is set to infinity.

Reply:
That's fine. But it also checks that the function outputs `0.24197072` and `0.10984782`. The source of magic numbers like these should be specified as a comment in the code.

Reply:
Please add the source of the numbers like 0.24197072 and 0.10984782 @zoj613 as a comment. Thanks!